contracts/RabbitHoleReceipt.sol

```diff
@@ -161,7 +162,7 @@ contract RabbitHoleReceipt is
         require(_exists(tokenId_), 'ERC721URIStorage: URI query for nonexistent token');
         require(QuestFactoryContract != IQuestFactory(address(0)), 'QuestFactory not set');
 
-        string memory questId = questIdForTokenId[tokenId_];
+        string storage questId = questIdForTokenId[tokenId_];
```

Comment:
`storage` is less gas than `memory` whys that?

Reply:
Actually I don't think this makes sense tbh. Undoing